### PR TITLE
styling for submenu

### PR DESCRIPTION
**Pull request by nadjak77**
Pull Request for Issue # .

### Summary of Changes
some styling for the submenu in the sidebar

### Expected result
the submenue has now a white background and some padding

### CODE REVIEW

**Comment by bembelimen** [comment]
I guess you mixed two PRs?

File: administrator/components/com_config/tmpl/component/default.php [hunk 1]
```diff
@@ -42,10 +42,9 @@
 
 		<?php // Begin Sidebar ?>
 		<div class="col-md-2" id="sidebar">
-            <button class="navbar-toggler options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
-                 <span class="burger-toggler-icon">
-                     <?php echo Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>
-                  </span>
+            <button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
+                 <span class="burger-toggler-icon"></span>
```
Comment (brianteeman) [inline administrator/components/com_config/tmpl/component/default.php]: aria hidden please on the font icon AND use standard fontawesome names

**Comment by brianteeman** [comment]
Please use standard fontawesome naming instead of inventing your own and dont forget to add the aria-hidden on the span with the icon in

File: administrator/components/com_config/tmpl/application/default.php [hunk 2]
```diff
@@ -31,12 +31,12 @@
 	<div class="row">
 		<!-- Begin Sidebar -->
 		<div id="sidebar" class="col-md-2">
-            <button class="navbar-toggler options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
-                 <span class="burger-toggler-icon">
+            <button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (brianteeman) [inline administrator/components/com_config/tmpl/application/default.php]:
```suggestion
            <button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>">
```

Use language strings not hardcoded english please

File: administrator/components/com_config/tmpl/application/default.php [hunk 3]
```diff
@@ -31,12 +31,12 @@
...
+                 <span class="burger-toggler-icon"></span>
```
Comment (brianteeman) [inline administrator/components/com_config/tmpl/application/default.php]:
```suggestion
                 <span class="burger-toggler-icon" aria-hidden="true"></span>
```

And use a regular icon class - either fa fa- or icon-

File: administrator/components/com_config/tmpl/component/default.php [hunk 4]
```diff
@@ -42,10 +42,9 @@
 
 		<?php // Begin Sidebar ?>
 		<div class="col-md-2" id="sidebar">
-            <button class="navbar-toggler options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
-                 <span class="burger-toggler-icon">
-                     <?php echo Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>
-                  </span>
+            <button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (brianteeman) [inline administrator/components/com_config/tmpl/component/default.php]:
```suggestion
            <button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>">
```

Dont hardcode to english

File: layouts/joomla/sidebars/submenu.php [hunk 5]
```diff
@@ -18,10 +18,9 @@
 <?php if ($displayData->displayMenu || $displayData->displayFilters) : ?>
 <div id="j-toggle-sidebar-wrapper">
 	<div id="sidebar" class="sidebar">
-		<button class="navbar-toggler options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
-			<span class="fas fa-align-justify">
+		<button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (brianteeman) [inline layouts/joomla/sidebars/submenu.php]:
```suggestion
		<button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>">
```

File: layouts/joomla/sidebars/submenu.php [hunk 6]
```diff
@@ -18,10 +18,9 @@
...
+			<span class="fas fa-align-justify"></span>
```
Comment (brianteeman) [inline layouts/joomla/sidebars/submenu.php]:
```suggestion
			<span class="fas fa-align-justify" aria-hidden="true"></span>
```

**Review by bembelimen — APPROVED** [review]

**Review by brianteeman — APPROVED** [review]
looks good to me now

**Pull request merged**